patterns/atoms/global/00-colors.mustache

```diff
@@ -0,0 +1,38 @@
+<ul class="sg-colors">
+	<li>
+		<span class="sg-swatch" style="background: #ff0000;"></span>
+		<span class="sg-label">#ff0000</span>
```

Comment:
Sorry, these four files were copied over from patternlab. I'm going to change them to correct pattern library format and resubmit.